lib/str_conv.mli

```diff
@@ -16,6 +16,7 @@ end
 
 module type S = sig
   type t
+  val of_substring : string -> int -> (t * int)
```

Comment:
What about labeling the `int` argument as `pos`? It's not really what it means otherwise.

Reply:
Make sens. I added a new commit.